packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.scss

```diff
@@ -0,0 +1,191 @@
+@mixin accumulatorSlideToNext {
```

Comment:
nit-pick :)
```suggestion
@mixin accumulator-slide-to-next {
```
in our style guide we mention that we are partially follow airbnb style guide for sass variables. Majority of all currently existing sass variables (mixins, keyframes) are following it:
<img width="1290" alt="Screenshot 2024-08-14 at 3 11 21 PM" src="https://github.com/user-attachments/assets/c6497536-4f53-4de5-9bb6-a6b645861a63">
*same for `slideToNext` and `accumulatorUnderlineAnimation` below